### PR TITLE
Set pfs-info infos via cli arguments

### LINKED ISSUE #479: Add CLI options for PFS info settings
Currently the information shown in the PFS info endpoint is not configurable. We need to add CLI args for them.

### PR DESCRIPTION
**Pull request by Dominik1999**
Closes #479

### CODE REVIEW

**Comment by Dominik1999** [comment]
Might be missing (I don't know if needed):
- test cli args itself (they are not required and take any string)
- too many arguments / instances in `ServiceApi` acc. to mypy. Error is now ignored, but we could put the three new args into a dict and change that ...?!

File: src/pathfinding_service/cli.py [hunk 1]
```diff
@@ -47,6 +47,19 @@
     help="Number of block confirmations to wait for",
 )
 @click.option("--enable-debug", default=False, is_flag=True, hidden=True)
+@click.option("--pfs-operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--pfs-settings",
```
Comment (palango) [inline src/pathfinding_service/cli.py]: I wonder if we should remove this for now, as it doesn't have a purpose right now.

File: src/pathfinding_service/cli.py [hunk 2]
```diff
@@ -47,6 +47,19 @@
...
+    default="Raiden Default Settings - bidirectional Dijkstra",
+    type=str,
+    help="Information on technical implementation",
+)
+@click.option(
+    "--pfs-message",
+    default="This is your favorite pathfinding service",
+    type=str,
+    help="Number of block confirmations to wait for",
```
Comment (palango) [inline src/pathfinding_service/cli.py]: This needs to be updated

File: src/pathfinding_service/cli.py [hunk 3]
```diff
@@ -47,6 +47,19 @@
...
+@click.option("--pfs-operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--pfs-settings",
+    default="Raiden Default Settings - bidirectional Dijkstra",
+    type=str,
+    help="Information on technical implementation",
+)
+@click.option(
+    "--pfs-message",
+    default="This is your favorite pathfinding service",
```
Comment (palango) [inline src/pathfinding_service/cli.py]: Can we make this more meaningful? Like `PFS for token network registry 0xdeadbeef`, or is this to technical?

File: tests/pathfinding/fixtures/api.py [hunk 4]
```diff
@@ -32,7 +32,13 @@ def api_sut(
     populate_token_network_case_1,  # pylint: disable=unused-argument
 ) -> Iterator[ServiceApi]:
     pathfinding_service_mock.address_to_reachability = address_to_reachability
-    api = ServiceApi(pathfinding_service_mock, one_to_n_address=Address(bytes([1] * 20)))
+    api = ServiceApi(
+        pathfinding_service_mock,
```
Comment (palango) [inline tests/pathfinding/fixtures/api.py]: Can you add the keyword in this line

File: tests/pathfinding/fixtures/api.py [hunk 5]
```diff
@@ -47,7 +53,12 @@ def api_sut_with_debug(
 ) -> Iterator[ServiceApi]:
     pathfinding_service_mock.address_to_reachability = address_to_reachability
     api = ServiceApi(
-        pathfinding_service_mock, one_to_n_address=Address(bytes([1] * 20)), debug_mode=True
+        pathfinding_service_mock,
```
Comment (palango) [inline tests/pathfinding/fixtures/api.py]: same here

File: src/pathfinding_service/cli.py (hunk 1 above)
Comment (Dominik1999) [reply src/pathfinding_service/cli.py]: yep that is a good idea

File: src/pathfinding_service/cli.py (hunk 2 above)
Comment (Dominik1999) [reply src/pathfinding_service/cli.py]: done

File: tests/pathfinding/fixtures/api.py (hunk 4 above)
Comment (Dominik1999) [reply tests/pathfinding/fixtures/api.py]: done

**Comment by palango** [comment]
Looks good, but let's wait till https://github.com/raiden-network/raiden/pull/4486 is green as well

File: tests/pathfinding/test_api.py [hunk 6]
```diff
@@ -264,20 +264,24 @@ def test_get_paths(api_url: str, addresses: List[Address], token_network_model:
...
             "chain_id": pathfinding_service_mock.chain_id,
-            "registry_address": to_checksum_address(pathfinding_service_mock.registry_address),
+            "registry_address": token_network_registry,
         },
-        "settings": "PLACEHOLDER FOR PATHFINDER SETTINGS",
         "version": pkg_resources.require("raiden-services")[0].version,
-        "operator": "PLACEHOLDER FOR PATHFINDER OPERATOR",
-        "message": "PLACEHOLDER FOR ADDITIONAL MESSAGE BY THE PFS",
+        "operator": "John Doe",
+        "message": "This is your favorite pfs for token network registry "
```
Comment (palango) [inline tests/pathfinding/test_api.py]: Oh, one problem here:

When I put a custom message this will still add the registry address. That shouldn't happen.

Comment (Dominik1999) [reply tests/pathfinding/test_api.py]: oh yes, you are right, will change

File: src/pathfinding_service/cli.py [hunk 7]
```diff
@@ -47,6 +47,13 @@
     help="Number of block confirmations to wait for",
 )
 @click.option("--enable-debug", default=False, is_flag=True, hidden=True)
+@click.option("--operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--info-message",
+    default="This is your favorite pfs for token network registry ",
```
Comment (palango) [inline src/pathfinding_service/cli.py]: I'd put this into a constant and use that in the `info` endpoint for the comparison as well

Comment (Dominik1999) [reply src/pathfinding_service/cli.py]: ok

**Pull request merged**